workers/unity/Assets/Playground/Scripts/Worker/ClientWorkerConnector.cs

```diff
@@ -1,12 +1,18 @@
 using System;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Core.Representation;
+using Improbable.Gdk.GameObjectCreation;
+using Improbable.Gdk.PlayerLifecycle;
+using Improbable.Gdk.TransformSynchronization;
+using Improbable.Worker.CInterop;
 using UnityEngine;
 
 namespace Playground
 {
     public class ClientWorkerConnector : WorkerConnector
     {
+        public const string UnityClient = "UnityClient";
```

Comment:
rename to `WorkerType` in here + other connectors?

Reply:
See comment [here](https://github.com/spatialos/gdk-for-unity/pull/1491#discussion_r560961852) 😄

Reply:
👍